base/doc/ltnews39.tex

```diff
@@ -190,7 +190,16 @@ \subsection{Warn if counter names are problematic}
 %
 \githubissue{823}
 
-
+\subsection{Extended information in \cs{listfiles}}
+
+The \cs{listfiles} command provides useful information when finding issues
+related to variation in package versions. However, this has to date relied on
+the information in the \cs{ProvidesPackage} line, or similar: that can be
+misleading if for example a file has been edited locally. We have now extended
+\cs{listfiles} to take an optional argument which will then include the MD5
+hash of each file (and the size of each file) in the \texttt{.log}.
```

Comment:
I think this should give an example, such as "So instead of \listfiles you can say \listfiles[hashes,sizes] to get the full information."

Reply:
Adjusted